### PR TITLE
[feat-4348]change the default token to a random number

### PR DESCRIPTION
**Pull request by aiceflower**
change the default token to a random number

### CODE REVIEW

**Comment by casionone** [comment]
waiting to be discussed

File: linkis-dist/package/conf/linkis.properties [hunk 1]
```diff
@@ -89,4 +89,12 @@ linkis.session.redis.password=test123
...
-wds.linkis.workspace.filesystem.path.check=true
\ No newline at end of file
+wds.linkis.workspace.filesystem.path.check=true
+
+#linkis token
+linkis.configuration.linkisclient.auth.token.value=BML-AUTH
+wds.linkis.client.common.tokenValue=BML-AUTH
+wds.linkis.bml.auth.token.value=BML-AUTH
+wds.linkis.context.client.auth.value=BML-AUTH
+wds.linkis.errorcode.auth.token=BML-AUTH
```
Comment (peacewong) [inline linkis-dist/package/conf/linkis.properties:99]: wds.linkis.filesystem.token.value need be add

Comment (peacewong) [reply linkis-dist/package/conf/linkis.properties:99]: wds.linkis.server.dsm.auth.token.value

File: linkis-public-enhancements/linkis-context-service/linkis-cs-client/src/test/java/org/apache/linkis/cs/client/utils/ContextClientConfTest.java [hunk 2]
```diff
@@ -33,7 +37,7 @@ public void constTest() {
     String hearBeatEnabled = ContextClientConf.HEART_BEAT_ENABLED().getValue();
 
     Assertions.assertEquals("Token-Code", contextClientAuthKey);
-    Assertions.assertEquals("BML-AUTH", contextClientAuthValue);
+    Assertions.assertEquals(bmlToken, contextClientAuthValue);
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-context-service/linkis-cs-client/src/test/java/org/apache/linkis/cs/client/utils/ContextClientConfTest.java:40]: can delete

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala [hunk 3]
```diff
@@ -38,8 +41,8 @@ object JobObserveActionTest extends App {
     .retryEnabled(false)
     .readTimeout(30000)
     .setAuthenticationStrategy(new TokenAuthenticationStrategy())
-    .setAuthTokenKey("BML-AUTH")
-    .setAuthTokenValue("BML-AUTH")
+    .setAuthTokenKey("Validation-Code")
```
[the remark this answers is not in the document]
Comment (casionone) [reply linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala:44]: make no difference

File: linkis-dist/package/conf/linkis.properties (hunk 1 above)
Comment (aiceflower) [reply linkis-dist/package/conf/linkis.properties:99]: modified

File: linkis-public-enhancements/linkis-context-service/linkis-cs-client/src/test/java/org/apache/linkis/cs/client/utils/ContextClientConfTest.java (hunk 2 above)
Comment (aiceflower) [reply linkis-public-enhancements/linkis-context-service/linkis-cs-client/src/test/java/org/apache/linkis/cs/client/utils/ContextClientConfTest.java]: modified

**Review by casionone — APPROVED** [review]

**Pull request merged**